pkg/kubelet/eviction/helpers.go

```diff
@@ -204,14 +255,52 @@ func memoryUsage(memStats *statsapi.MemoryStats) *resource.Quantity {
 	return resource.NewQuantity(usage, resource.BinarySI)
 }
 
-// podUsage aggregates usage of compute resources.
-// it supports the following memory and disk.
-func podUsage(podStats statsapi.PodStats) (api.ResourceList, error) {
+// localVolumeNames returns the set of volumes for the pod that are local
```

Comment:
nit: mention that these volumes consume disk.

Ideally, I'd argue that we should let the metrics summary API provide more information about what resources a volume consumes. For example, a tmpfs based empty dir consumes memory, and a regular empty dir consumes local disk and an nfs volume consumes remote storage, etc. Hard coding the list here is fine, but is probably not the cleanest solution. WDYT?